api/resource/resource.proto

```diff
@@ -92,3 +92,30 @@ message WatchResponse {
   Resource definition = 3;
   Resource resource = 4;
 }
+
+message URL {
```

Comment:
should we move this to e.g. `api/resource/common.proto`, as whole `resource.proto` is deprecated now